.travis.yml

```diff
@@ -59,6 +62,31 @@ jobs:
       script:
         - bash .travis.sh script
         - bash .travis.sh test
+    - name: azure
+      env:
+        - K8_PROVIDER=azure
+        - CCO_INTERACTIVE_CI=interactive.yaml
+        - HELM_VERSION=v2.16.1
+        - KUBECONFIG=kubeconfig_terraform-cco
+      before_install: pip install -e .
+      install: bash .travis.sh install-tools
+      script:
+        - az login --service-principal -u $ARM_CLIENT_ID -p $ARM_CLIENT_SECRET --tenant $ARM_TENANT_ID
```

Comment:
check out the init_cluster.sh script in terraform/aws - perhaps you could follow that pattern?